spec/Chassis/Chassis.vspec

```diff
@@ -178,3 +178,11 @@ Brake.PedalPosition:
   unit: percent
   description: Brake pedal position as percent. 0 = Not depressed. 100 = Fully depressed.
 
+Brake.IsEmergencyBraking:
+  datatype: boolean
+  type: sensor
+  description: Indicates if emergency braking initiated by driver is detected.
+               True = Emergency braking detected. False  = Emergency braking not detected.
+  comment: Detection of emergency braking can trigger Emergency Braking Assist (EBA) to engage.
```

Comment:
```suggestion
  comment: Detection of emergency braking can trigger Emergency Brake Assist (EBA) to engage.
```

as in https://github.com/boschglobal/vehicle_signal_specification/blob/3e839f4bd4551de4dadb350aa42195600f2797de/spec/ADAS/ADAS.vspec#L229